contrib/agent/src/integration-test/java/io/opencensus/contrib/agent/instrumentation/ThreadInstrumentationIT.java

```diff
@@ -115,18 +114,15 @@ public void execute(Runnable command) {
             // been wrapped in a different context (by automatic instrumentation of
             // Executor#execute), that context will be attached when executing the Runnable.
             Context context2 = Context.current().withValue(KEY, "wrong context");
-            // Work around findbugs warning. Context.attach() is marked as @CheckReturnValue so we
-            // need to check the return
-            // value here, otherwise findbugs will fail.
-            Preconditions.checkNotNull(context2.attach(), "context2.attach()");
+            Context context3 = context2.attach();
             Thread thread = new Thread(command);
             thread.start();
             try {
               thread.join();
             } catch (InterruptedException ex) {
               Thread.currentThread().interrupt();
             }
-            context2.detach(context);
+            context2.detach(context3);
```

Comment:
`attach()` should always be immediately followed by a `try-finally` that calls `detach()` in `finally`.
```java
Context context3 = context2.attach();
try {
  ...
} finally {
  context2.detach(context3)
}
```

Reply:
I see, updated.